ReactAndroid/src/main/java/com/facebook/react/uimanager/TouchTargetHelper.java

```diff
@@ -144,8 +146,12 @@ private static boolean isTransformedTouchPointInView(
       localX = localXY[0];
       localY = localXY[1];
     }
-    if ((localX >= 0 && localX < (child.getRight() - child.getLeft()))
-        && (localY >= 0 && localY < (child.getBottom() - child.getTop()))) {
+    Rect hitSlopRect = new Rect();
+    if (child instanceof ReactViewGroup && ((ReactViewGroup) child).getHitSlopRect() != null) {
```

Comment:
Can you extract `getHitSlopRect` to an interface so that other views (such as Text elements) can implement hitSlop too if we need it?